integration/terraform/ec2/linux/main.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_instance" "integration-test" {
+  ami                    = data.aws_ami.latest.id
+  instance_type          = var.ec2_instance_type
+  key_name               = var.key_name
+  iam_instance_profile   = var.iam_instance_profile
+  vpc_security_group_ids = var.vpc_security_group_ids
+  provisioner "remote-exec" {
+    inline = [
+      "cloud-init status --wait",
+      "echo clone, build, and install agent",
+      "git clone ${var.github_repo}",
+      "cd amazon-cloudwatch-agent",
+      "git reset --hard ${var.github_sha}",
+      "make clean build ${var.package}",
+      "cd build/bin/linux/amd64",
```

Comment:
This may feel like an obvious question, but will this always work for all Linux installs we do in the GitHub actions?

Reply:
duplicate